File: docs/sharepoint/codesnippet/CSharp/testsitedef/sitedefinition/visualwebpart1/visualwebpart1usercontrol.ascx.cs
using System;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;

namespace TestSiteDef.SiteDefinition.VisualWebPart1
{
    public partial class VisualWebPart1UserControl : UserControl
    {
        protected void Page_Load(object sender, EventArgs e)
        {         
        }

        //<Snippet1>
        protected void btnSubmit_Click(object sender, EventArgs e)
        {            
            lblName.Text = tbName.Text;
        }
        //</Snippet1>

    }
}

File: docs/sharepoint/codesnippet/VisualBasic/testsitedefvb/sitedefinition/visualwebpart1/visualwebpart1usercontrol.ascx.vb
Imports System
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts

Partial Public Class VisualWebPart1UserControl
    Inherits UserControl

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load

    End Sub

    '<Snippet1>
    Protected Sub btnSubmit_Click(ByVal sender As Object, ByVal e As EventArgs)
        lblName.Text = tbName.Text
    End Sub
    '</Snippet1>

End Class
